packages/transaction-controller/src/utils/feature-flags.ts

```diff
@@ -8,6 +8,22 @@ export const FEATURE_FLAG_TRANSACTIONS = 'confirmations_transactions';
 export const FEATURE_FLAG_EIP_7702 = 'confirmations_eip_7702';
 
 const DEFAULT_BATCH_SIZE_LIMIT = 10;
+const DEFAULT_GAS_ESTIMATE_FALLBACK_BLOCK_PERCENT = 35;
+
+type GasEstimateFallback = {
+  /**
+   * The fallback gas estimate for a transaction.
+   * This value is either a fixed hexadecimal number or a percentage multiplier.
+   */
+  value: number | Hex;
```

Comment:
Can we consider setting this value as a decimal number even when `isFixedGas` is true? This makes it easier to configure on launch darkly, and the code can easily handle the hex conversion.

Reply:
Agree that a `number` is much easier to input.